autogpt_platform/frontend/sentry.edge.config.ts

```diff
@@ -39,4 +39,8 @@ Sentry.init({
   debug: false,
 
   enableLogs: true,
+  integrations: [
+    Sentry.captureConsoleIntegration(),
```

Comment:
Captures console errors + extra data about errors

Reply:
in the edge collect the conosle (because we aren't great at not using it right now) and extra error data for diagnostics when one occurs